src/components/searchBar/SearchBar.tsx

```diff
@@ -0,0 +1,52 @@
+import { InputAdornment, styled } from '@mui/material'
+import { useState } from 'react'
+
+import { Search } from '@mui/icons-material'
+import { StyledTextField } from '../inputs/TextField'
+
+interface ISearchBar {
+  value: string
+  getSearchKeyword: (keyword: string) => void
+}
+
+const SearchBar = ({ value, getSearchKeyword }: ISearchBar) => {
+  const [focused, setFocused] = useState(false)
+
+  return (
+    <StyledTextInput
+      value={value}
+      focused={focused}
+      variant="outlined"
+      placeholder={focused ? 'Search...' : 'Search'}
+      size="small"
+      InputProps={{
+        startAdornment: (
+          <InputAdornment
+            position="start"
+            sx={{
+              cursor: 'pointer',
+            }}
+          >
+            <Search />
+          </InputAdornment>
+        ),
+      }}
```

Comment:
The icon used for this component does not match the one specified by Copilot in their style guide, nor is it proportional to the size of the search box

![image](https://github.com/copilot-platforms/tasks-app/assets/59971845/1c6b6436-9552-453b-bb50-c352c5bb173d)

![image](https://github.com/copilot-platforms/tasks-app/assets/59971845/85af7ce5-2d74-44e3-bc45-9ac0c62bc0bd)

Reply:
Please refer to the figma style guide section or drill down to the icon from the design and download / copy it as SVG

Reply:
@arpandhakal Seems like you've used the MUI icon here. Please use the same png icon as being used in the Profile Manager app which is consistent throughout the Copilot dashboard. You can copy that icon in the `/icons` folder and export it from `/icons/index.tsx`.